tests/TestClient.h

```diff
@@ -16,9 +16,10 @@ class TestClient : public QXmppClient
 {
     Q_OBJECT
 public:
-    TestClient(bool enableDebug = false)
+    TestClient(bool enableDebug = false, bool disableAutoReset = false)
```

Comment:
That makes it easier to read/understand:
```suggestion
    TestClient(bool enableDebug = false, bool enableAutoReset = true)
```